site/pages/institution-landing-en.hbs

```diff
@@ -0,0 +1,213 @@
+---
+{
+"title": "Institution-landing-page",
+"language": "en",
+"altLangPrefix": "institution-landing",
+"pageType": "institution-landing",
+"pageclass": "page-type-nav",
+"breadcrumb": [
+{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" },
+{ "title": "Departments & agencies", "link": "https://www.canada.ca/en/government/dept.html" }
+],
+"dateModified": "2020-02-07",
+"share": "true",
+"noMainContainer": true,
+"followUsLayout": "inline"
+}
+---
+
+<div class="provisional gc-institutional-bg mrgn-tp-md"
+	data-bgimg="https://design.canada.ca/coded-layout/images/ip-cover-image-1200x726.jpg">
+	<div class="container">
+		<div class="row">
+			<div class="col-md-7">
+				<h1 property="name" id="wb-cont">[Institution name]</h1>
+				<p>Short description of the institution’s mandate.</p>
+				<a class="provisional btn btn-call-to-action" href="#">Super task button [optional]</a>
+			</div>
+			<div class="clearfix"></div>
+		</div>
+	</div>
+</div>
+{{!-- Featured Link --}}
+<div class="provisional gc-featured-lnk">
+	<div class="container">
+		<div class="row">
+			<div class="mrgn-tp-md mrgn-bttm-md col-xs-12">
+				<h2 class="wb-inv">Featured link</h2>
+				<p class="pull-left mrgn-bttm-0"><a class="stretched-link text-white" href="#">Featured link
+						[optional]</a></p>
+			</div>
+		</div>
+	</div>
+</div>
+
+{{!-- Most Requested --}}
+<section class="well well-sm brdr-0 mrgn-tp-0">
+	<div class="container">
+		<div class="row">
+			<small>
```

Comment:
Small element are phrasing content, it should not contain block content. It will need to be replaced. Probably the best solution would be to create a gc-mostrequested provisional components.